### PR TITLE
Fix the Xposed Module notification repeated push problem and the other problem

### PR DESCRIPTION
**Pull request by fankes**
Fix the bug of repeated push notification of module activation in the previous version, and fix the bug of repeated push notification after the module is uninstalled

### CODE REVIEW

File: daemon/src/main/java/org/lsposed/lspd/service/LSPManagerService.java [hunk 1]
```diff
@@ -253,11 +253,27 @@ private static int getAutoIncrementNotificationId() {
 
     private static int pushAndGetNotificationId(String modulePackageName, int moduleUserId) {
         var idKey = getNotificationIdKey(modulePackageName, moduleUserId);
-        var idValue = getAutoIncrementNotificationId();
-        notificationIds.putIfAbsent(idKey, idValue);
+        var idValue = notificationIds.get(idKey);
```
Comment (vvb2060) [inline daemon/src/main/java/org/lsposed/lspd/service/LSPManagerService.java:256]: computeIfAbsent

**Review by vvb2060 — CHANGES_REQUESTED** [review]

File: daemon/src/main/java/org/lsposed/lspd/service/LSPManagerService.java (hunk 1 above)
Comment (fankes) [reply daemon/src/main/java/org/lsposed/lspd/service/LSPManagerService.java:256]: image

Comment (fankes) [reply daemon/src/main/java/org/lsposed/lspd/service/LSPManagerService.java:256]: resolve it

**Comment by fankes** [comment]
@vvb2060 What's the next step?

**Comment by Howard20181** [comment]
> @vvb2060 What's the next step?

Please check if it works properly in a multi-user environment

**Comment by fankes** [comment]
> > @vvb2060 What's the next step?
> 
> Please check if it works properly in a multi-user environment

What is the proof factor?

**Comment by Howard20181** [comment]
> 

Make sure there are multiple users (or one user with a work-profile)
Install an application using adb
Uninstall it

**Comment by fankes** [comment]
> Make sure there are multiple users (or one user with a work-profile)
> Install an application using adb
> Uninstall it

How do you prove that it works properly? What are the specific operating procedures and your acceptance criteria?

**Comment by Howard20181** [comment]
> > Make sure there are multiple users (or one user with a work-profile)
> > Install an application using adb
> > Uninstall it
> 
> How do you prove that it works properly? What are the specific operating procedures and your acceptance criteria?

Now
The above steps will pop up 2 notifications
Then after uninstallation only one will be cancel

The expectation is that both notifications will be cancelled

**Comment by fankes** [comment]
> > > Make sure there are multiple users (or one user with a work-profile)
> > > Install an application using adb
> > > Uninstall it
> > 
> > How do you prove that it works properly? What are the specific operating procedures and your acceptance criteria?
> 
> Now
> The above steps will pop up 2 notifications
> Then after uninstallation only one will be cancel
> 
> The expectation is that both notifications will be cancelled

Does the modified code still have this problem?

This problem appeared in the last ci version, but this pr is to fix this problem, can you test it again?

**Comment by Howard20181** [comment]
> > > > Make sure there are multiple users (or one user with a work-profile)
> > > > Install an application using adb
> > > > Uninstall it
> > > 
> > > 
> > > How do you prove that it works properly? What are the specific operating procedures and your acceptance criteria?
> > 
> > 
> > Now
> > The above steps will pop up 2 notifications
> > Then after uninstallation only one will be cancel
> > The expectation is that both notifications will be cancelled
> 
> Does the modified code still have this problem?
> 
> This problem appeared in the last ci version, but this pr is to fix this problem, can you test it again?

The above test results are based on this PR

**Comment by fankes** [comment]
> > > > > Make sure there are multiple users (or one user with a work-profile)
> > > > > Install an application using adb
> > > > > Uninstall it
> > > > 
> > > > 
> > > > How do you prove that it works properly? What are the specific operating procedures and your acceptance criteria?
> > > 
> > > 
> > > Now
> > > The above steps will pop up 2 notifications
> > > Then after uninstallation only one will be cancel
> > > The expectation is that both notifications will be cancelled
> > 
> > 
> > Does the modified code still have this problem?
> > This problem appeared in the last ci version, but this pr is to fix this problem, can you test it again?
> 
> The above test results are based on this PR

It should be fixed, please check again

**Review by Howard20181 — APPROVED** [review]

**Pull request merged**